lime/lime_tabular.py

```diff
@@ -157,11 +157,10 @@ def __init__(self, training_data, training_labels=None, feature_names=None,
             kernel_width = np.sqrt(training_data.shape[1]) * .75
         kernel_width = float(kernel_width)
 
-        def kernel(d):
-            return np.sqrt(np.exp(-(d ** 2) / kernel_width ** 2))
-
         self.feature_selection = feature_selection
-        self.base = lime_base.LimeBase(kernel, verbose)
+        self.base = lime_base\
```

Comment:
I'm not a fan of the readability here, how the command spans three lines.

Won't matter if this change is reverted.

Nudge @proinsias to continue with this PR.